charts/prism/templates/computations-deployment.yaml

```
@@ -1,12 +1,29 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
 metadata:
   name: {{ .Release.Name }}-computations
 spec:
-  replicas: {{ .Values.defaults.replicaCount }}
+  replicas: 1
+  strategy:
+    {{- if .Values.env.prod }}
+    canary:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)